integrations/clickup/.port/spec.yaml

```diff
@@ -0,0 +1,19 @@
+description: clickup integration for Port Ocean
+icon: Clickup
+docs: ""
+features:
+  - type: exporter
+    section: Project management
+    resources:
+      - kind: team
+      - kind: project
+      - kind: issues
```

Comment:
change this to singular `issue` be sure to make the necessary changes in the ObjecKind